ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/traffic-policies.j2

```diff
@@ -149,6 +149,36 @@ No Match condition configured.
 {%             endif %}
 
 {%         endfor %}
+{%     endif %}
+{# Traffic-Policy Interfaces #}
+{%     set traffic_policy_interfaces = [] %}
+{%     for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
+{%         if ethernet_interfaces[ethernet_interface].traffic_policy.input is arista.avd.defined or ethernet_interfaces[ethernet_interface].traffic_policy.output is arista.avd.defined %}
+{%             do traffic_policy_interfaces.append(ethernet_interface) %}
+{%         endif %}
+{%     endfor %}
+{%     for port_channel_interface in port_channel_interfaces | arista.avd.natural_sort %}
+{%         if port_channel_interfaces[port_channel_interface].traffic_policy.input is arista.avd.defined or port_channel_interfaces[port_channel_interface].traffic_policy.output is arista.avd.defined %}
+{%             do traffic_policy_interfaces.append(port_channel_interface) %}
+{%         endif %}
+{%     endfor %}
+{%     if traffic_policy_interfaces | length > 0 %}
```

Comment:
if traffic_policy_eth_interfaces | length > 0 or traffic_policy_po_interfaces | length > 0

Reply:
Fixed according to Claus' suggestion.